lib/mappings.js

```diff
@@ -1,17 +1,33 @@
 'use strict';
 
-const Hoek = require('hoek');
+const Joi = require('joi');
 
 const internals = {};
 
+const multiReturn = (condition, errMsg, val) => {
```

Comment:
If you could put utilities, etc. on `internals` and move `module.exports` towards the top of the file, that would be awesome!

Reply:
Will do